app/include/zmk/events/behavior_binding_event.h

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2025 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#pragma once
+
+#include <zephyr/kernel.h>
+#include <zmk/event_manager.h>
+#include <zephyr/sys/util.h>
+
+typedef uint16_t zmk_behavior_local_id_t;
+
+struct zmk_behavior_binding {
+#if IS_ENABLED(CONFIG_ZMK_BEHAVIOR_LOCAL_IDS_IN_BINDINGS)
+    zmk_behavior_local_id_t local_id;
+#endif // IS_ENABLED(CONFIG_ZMK_BEHAVIOR_LOCAL_IDS_IN_BINDINGS)
+    const char *behavior_dev;
+    uint32_t param1;
+    uint32_t param2;
+};
+
+enum trigger_type { PRESS, RELEASE, SENSOR };
```

Comment:
The enum name *and* the entries should get proper namespace prefixies, e.g.:

```suggestion
enum zmk_behavior_trigger_type {
    ZMK_BEHAVIOR_TRIG_TYPE_PRESS,
    ZMK_BEHAVIOR_TRIG_TYPE_RELEASE,
    ZMK_BEHAVIOR_TRIG_TYPE_SENSOR,
};
```